lambdas/preview/index.py

```diff
@@ -9,12 +9,15 @@
 from jsonschema import Draft4Validator, ValidationError
 from nbconvert import HTMLExporter
 import nbformat
-import pandas as pd
-import pyarrow as pa
+import pyarrow.parquet as pq
 import requests
 
-WEB_ORIGIN = os.environ['WEB_ORIGIN']
 
+ALLOWED_ORIGINS = [
+    'http://localhost:3000',
+    'http://localhost:3000/',
```

Comment:
I looked up the spec; there should be no trailing slash, so let's remove this one. (Though we could add a sanity check for `os.environ.get('WEB_ORIGIN')`, to make sure it doesn't have one.)